docs/src/_theme/DocSidebar/index.jsx

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
```

Comment:
any reason why the folder in named `_theme` instead of `theme`?

Reply:
To prevent it from overriding. I changed customization methods part-way through since the `swizzle` approach was proving tedious (I was having to make many small changes in different component files), but I'm keeping these two components here because I'd already started altering them. Shortly I'll salvage the changes from here and move them in to the theme